### PR TITLE
Hiding members and issue tracker cards when no data is present

### PR DESCRIPTION
**Pull request by jmpainter**
- ProjectSummary.js - adding conditional rendering for members and issue tracker cards if not data is present

If this looks ok, I can add tests to ProjectSummary.test.js

### CODE REVIEW

**Comment by mattwr18** [comment]
@jmpainter it is failing the build on `standard` errors, can you please run `yarn fix` and push it back up?

**Comment by jmpainter** [comment]
I ran yarn fix and pushed it up. Will make sure to do in the future.

**Comment by jmpainter** [comment]
I added tests in ProjectSummary.test.js

File: src/components/ProjectSummary.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ import '../assets/ProjectSummary.css'
 
 const ProjectSummary = props => {
   let { project } = props
+  console.log(project)
```
Comment (joaopapereira) [inline src/components/ProjectSummary.js]: we should remove this console log

File: src/components/ProjectSummary.js [hunk 2]
```diff
@@ -25,87 +26,91 @@ const ProjectSummary = props => {
...
-                  <Card fluid raised className='project-info-card'>
-                    <Card.Content>
-                      <Card.Header>Connected on</Card.Header>
-                      <Card.Description className='project-info-card-description'>
-                        {project.sourceRepositories.map(repo => {
-                          let array = repo.url.split('/')
-                          let repoName = array[array.length - 1] || array[array.length - 2]
-                          return (
-                            <Fragment key={repo.id}>
+                {(project.pivotaltracker_url || project.sourceRepositories.length > 0 || project.slack_channel_name) ? (
```
Comment (joaopapereira) [inline src/components/ProjectSummary.js:28]: it is starting to look like a new component is needed here to make it more readable. Nevertheless I am ok merging it like this and refactor it later

**Review by joaopapereira — CHANGES_REQUESTED** [review]
LGTM, please just remove the console log and we should be fine to merge it

**Comment by jmpainter** [comment]
I removed the console.log from ProjectSummary.js

**Comment by mattwr18** [comment]
great job @jmpainter!!

**Review by joaopapereira — APPROVED** [review]
LGTM

**Pull request merged**